addons/payment_fix/models/account_payment_register.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+
+
+class AccountPaymentRegister(models.TransientModel):
+    _inherit = 'account.payment.register'
+
+    # == Business fields ==
+    payment_token_id = fields.Many2one(
+        comodel_name='payment.token',
+        string="Saved payment token",
+        store=True, readonly=False,
+        compute='_compute_payment_token_id',
+        domain='''[
+            (payment_method_code == 'electronic', '=', 1),
+            ('company_id', '=', company_id),
+            ('acquirer_id.capture_manually', '=', False),
+            ('acquirer_id.journal_id', '=', journal_id),
+            ('partner_id', 'in', suitable_payment_token_partner_ids),
+        ]''',
+        help="Note that tokens from acquirers set to only authorize transactions (instead of capturing the amount) are "
+             "not available.")
```

Comment:
heu? ok? why not the capturing _manually ones? was it like that before?

Reply:
Copy pasted from:
https://github.com/odoo/odoo/blob/14.0/addons/payment/models/account_payment.py#L22